nav2_bt_navigator/include/nav2_bt_navigator/navigators/navigate_fleet_nodes.hpp

```diff
@@ -0,0 +1,130 @@
+#pragma once
+
+#include <string>
+#include <vector>
+#include <memory>
+#include "rclcpp/rclcpp.hpp"
+#include "rclcpp_action/rclcpp_action.hpp"
+#include "geometry_msgs/msg/pose_stamped.hpp"
+#include "nav2_bt_navigator/navigator.hpp"
+#include "cmr_msgs/action/navigate_fleet_nodes.hpp"
+#include "nav_msgs/msg/path.hpp"
+#include "nav2_util/robot_utils.hpp"
+#include "nav2_util/geometry_utils.hpp"
+#include "nav2_util/odometry_utils.hpp"
+#include "nav_2d_utils/conversions.hpp"
+
+namespace nav2_bt_navigator
+{
+
+/**
+ * @class NavigateFleetNodesNavigator
+ * @brief A navigator for navigating to a a bunch of intermediary poses
+ */
+class NavigateFleetNodesNavigator
+  : public nav2_bt_navigator::Navigator<cmr_msgs::action::NavigateFleetNodes>
+{
+public:
+  using ActionT = cmr_msgs::action::NavigateFleetNodes;
+  typedef std::vector<geometry_msgs::msg::PoseStamped> Goals;
+  typedef std::vector<std::string> Actions;
```

Comment:
I'd prefer having a single way of defining aliases, `using` ideally

Reply:
`using` is compatible with templates and `typedef` isn't so I guess I'd tend to agree

Reply:
this is original code, from the two builtin navigators, navigate_to_pose, and navigate_through_poses.
I suppose the typedef/using selection was made for extactly that reason of differentiating between template use and not. 

We will be free to change this when they have a plugin mechanism.  I saw a PR for that recently